manual/manual.tex

```diff
@@ -103,7 +103,8 @@
 
   \vfill
 
-  \copyright~{\commityear} General Dynamics C4 Systems.\\
+  \copyright~{2014} General Dynamics C4 Systems.\\
+  \copyright~{\commityear} seL4 Foundation.\\
 
   \textsc{All rights reserved}.
```

Comment:
We should probably update this to the license the manual is provided under. Ideally this would be CC-BY-SA-4.0, but to re-release under that license we'd need the consent of all copyright holders, which is probably too hard by now. So I guess it has to remain GPL-2.0-only as defined in the SPDX header.

We could say:
```suggestion
  \textsc{All rights reserved}. You may use this manual and its sources under the terms of the GPL license, version 2.
```